### DISCUSSION TITLE
ScriptReference/SceneManagement.Scene-path

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#334] [Notes]
# ScriptReference/SceneManagement.Scene-path

https://docs.unity3d.com/ScriptReference/SceneManagement.Scene-path.html

**Comment by houkanshan** [comment]
Could be null if the scene is not loaded before, especially when you use it in editor.  
Use this instead:  
[`SceneUtility.GetScenePathByBuildIndex(0);`](https://docs.unity3d.com/ScriptReference/SceneManagement.SceneUtility.html)

**Comment by FreyaHolmer** [comment]
This is an empty string for new/untitled/unsaved scenes that don't have a corresponding scene asset saved yet, which is a useful way to check that very thing, since `IsValid()` still returns `true` for scenes without a corresponding scene asset
